src/api/claims.ts

```diff
@@ -4,10 +4,464 @@ import { AuthRequest } from '../lib/auth';
+        nestedMetadata: imageData.metadata.metadata 
+      }
+    };
+  }
+  
+  // Check for other unexpected nested structures in metadata
+  const metadataKeys = Object.keys(imageData.metadata);
+  const suspiciousNesting = metadataKeys.filter(key => 
+    typeof imageData.metadata[key] === 'object' && 
+    imageData.metadata[key] !== null &&
+    (key === 'filename' || key === 'contentType' || key === 'base64' || key === 'effectiveDate')
+  );
+  
+  if (suspiciousNesting.length > 0) {
+    return { 
+      isValid: false, 
+      error: `Image ${index}: metadata contains unexpected nested structures for fields: [${suspiciousNesting.join(', ')}]. These should be top-level fields, not nested in metadata.`,
+      details: { 
+        suspiciousNesting,
+        metadataStructure: imageData.metadata
+      }
+    };
+  }
+  
+  // Validate effectiveDate
+  if (imageData.effectiveDate) {
+    const dateValue = new Date(imageData.effectiveDate);
+    if (isNaN(dateValue.getTime())) {
+      return { 
+        isValid: false, 
+        error: `Image ${index}: 'effectiveDate' must be a valid date. Received: ${imageData.effectiveDate}`,
+        details: { receivedEffectiveDate: imageData.effectiveDate }
+      };
+    }
+  }
+  
+  console.log(`Image ${index} validation passed`);
+  return { isValid: true };
+}
+
+function validateRequestBody(body: any): { isValid: boolean; errors: string[]; validationDetails: any[] } {
+  const errors: string[] = [];
+  const validationDetails: any[] = [];
+  
+  // Required fields
+  if (!body.subject || typeof body.subject !== 'string') {
+    errors.push('Subject is required and must be a string');
+    validationDetails.push({
+      field: 'subject',
+      issue: 'missing_or_invalid_type',
+      received: { type: typeof body.subject, value: body.subject },
+      expected: 'string'
+    });
+  }
+  
+  if (!body.claim || typeof body.claim !== 'string') {
+    errors.push('Claim is required and must be a string');
+    validationDetails.push({
+      field: 'claim',
+      issue: 'missing_or_invalid_type',
+      received: { type: typeof body.claim, value: body.claim },
+      expected: 'string'
+    });
+  }
+  
+  // Optional string fields validation
+  const optionalStringFields = ['object', 'sourceURI', 'statement', 'aspect', 'unit', 'issuerId'];
+  optionalStringFields.forEach(field => {
+    if (body[field] !== undefined && body[field] !== null && typeof body[field] !== 'string') {
+      errors.push(`${field} must be a string if provided`);
+      validationDetails.push({
+        field,
+        issue: 'invalid_type',
+        received: { type: typeof body[field], value: body[field] },
+        expected: 'string'
+      });
+    }
+  });
+  
+  // Enum validation
+  if (body.howKnown && !['FIRST_HAND', 'SECOND_HAND', 'WEB_DOCUMENT', 'VERIFIED_LOGIN', 'BLOCKCHAIN', 'SIGNED_DOCUMENT', 'PHYSICAL_DOCUMENT', 'INTEGRATION', 'RESEARCH', 'OPINION', 'OTHER'].includes(body.howKnown)) {
```

Comment:
this should pull from the enum - otherwise we can get out of sync.  Can you fix this in a follow on please?